client/app/telecom/pack/slots/emailPro/detail/pack-emailPro-detail.html

```diff
@@ -0,0 +1,139 @@
+                    <dd class="service-item">
+                        <a href="{{::$ctrl.configuration.webmailUrl}}"
+                           data-translate-attr="{title:'email_pro_detail_open_webmail'}"
+                           title="{{::$ctrl.configuration.webmailUrl}} {{::'common_newtab' | translate}}"
+                           rel="noopener"
+                           target="_blank">
+                            <span data-ng-bind="$ctrl.configuration.webmailUrl"></span>
+                            <span class="ovh-font ovh-font-newtab" aria-hidden="true"></span>
+                        </a>
+                    </dd> <!-- END OF WEBMAIL -->
+
+                    <!-- GUIDE -->
+                    <dt class="service-title" data-translate="email_pro_detail_need_help"></dt>
+                    <dd class="service-item">
+                        <a href="{{::$ctrl.guideUrl}}"
+                           data-translate-attr="{title:'email_pro_detail_guide'}"
+                           title="{{::$ctrl.guideUrl}} {{::'common_newtab' | translate}}"
+                           rel="noopener"
+                           target="_blank">
+                            <span data-translate="email_pro_detail_guide"></span>
+                            <span class="ovh-font ovh-font-newtab" aria-hidden="true"></span>
+                        </a>
+                    </dd> <!-- END OF GUIDE -->
+
+                </dl>
+            </div>
+        </div>
+
+        <div class="row"
+             data-ng-if="!$ctrl.loading">
+            <!-- Change password -->
+            <div class="col-md-6 col-xs-12 widget-presentation">
+                <h2 class="widget-presentation-title"
+                    data-translate="email_pro_detail_client_change_password">
+                </h2>
+                <form name=passwordChange data-ng-submit="$ctrl.changePassword()">
+                    <div
+                        data-ovh-password
+                        data-ovh-pwd="$ctrl.password"
+                        data-ovh-pwd-confirm="ctrl.account.passwordConfirmation">
+                    </div>
+
+                    <button class="btn btn-primary"
+                            type="submit"
+                            data-ng-disabled="passwordChange.$invalid || $ctrl.changingPassword">
+                            <span data-translate="email_pro_detail_client_change_password_submit"></span>
+                            <oui-spinner data-ng-if="$ctrl.changingPassword"
+                                         data-size="s">
+                            </oui-spinner>
+                    </button>
+                </form>
+            </div>
+
+            <div class="col-md-6 col-xs-12 widget-presentation">
+                <h2 class="widget-presentation-title"
+                    data-translate="email_pro_detail_client_delete_account">
+                </h2>
+
+                <div class="alert alert-warning"
```

Comment:
Could be a [`<oui-message>`](http://master.ui-kit.ovh/#!/oui-angular/message) here I guess.